tap/tests/test_tracker.py

```diff
@@ -87,7 +92,8 @@ def test_individual_report_has_no_plan_when_combined(self):
         self.assertTrue('Look ma' in report)
         self.assertFalse('1..' in report)
 
-    def test_combined_results_in_one_file(self):
+    @mock.patch('tap.tracker.ENABLE_VERSION_13', False)
+    def test_combined_results_in_one_file_tappy_version12(self):
```

Comment:
It's a *TAP* version so these method names should probably end with `_tap_version_1*`.